src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.VisualBasic.DesignTime.targets

```diff
@@ -40,7 +40,9 @@
                           ClassDesigner;
                           SharedProjectReferences;" />
 
-      <ProjectCapability Include="EnableMyApplication" Condition="$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)','5.0')) And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(UseWindowsForms)' == 'true'" />
+      <ProjectCapability Include="EnableMyApplication" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'"/>
```

Comment:
Does this work? MyApplication wasn't enabled for SDK-style projects targeting .NET Framework before.

Reply:
The `EnableMyApplication` capability has been added to legacy as well.

Reply:
There's a couple of combos:

1) Legacy with WPF
2) Legacy with WinForms
3) CPS (.NET Framework) with WPF
4) CPS (.NET Framework) with WinForms
5) CPS (.NET Core) with WPF
6) CPS (.NET Core) with WinForms

What exactly is the state for all these? And what hit the path guarded by EnableMyApplication?

Reply:
Previous to #6486 only 1 and 2 in that list would use MyApplication features.

Reply:
This PR doesn't take WPF into account.

I suggest merging this as-is to unblock dogfooders. If there's any outstanding action required, we can file an issue. It should be easy to create a follow up PR.